src/Main.elm

```diff
@@ -39,12 +52,15 @@ init _ =
 type alias Model =
     { rhymes : RhymeResult
     , phrases : PhraseResult
+    , wordInput : String
```

Comment:
Since the debouncer also captures this value, thoughts on removing storing this at the top-level?

Reply:
I _think_ you're right. I thought I needed it because coupling the model's `word` with what the user is typing would have been problematic, but I think you also mentioned that it's stored in the debouncer so I can get at the new word that way also.